lib/progress_bar.rb

```diff
@@ -11,10 +11,9 @@ def initialize(total_size)
     # character used to fill the progress bar, one of the box-drawing character in unicode
     @bar_char = "\u2501"
 
-    # color scheme of progress bar, can be changed
-    # see color.rb for more available colors
-    @bar_front_color = :lightcyan
-    @bar_bg_color = :gray
+    # color scheme of progress bar (in ANSI color codes), can be changed
+    @bar_front_color = [96, 0]
+    @bar_bg_color = [90, 0]
```

Comment:
Why the change?  What is wrong with the more descriptive symbol notation?

Reply:
Since the `90-99` color code does not exist in `color.rb`

Reply:
(thinking what should they name if we decided to add them to `color.rb` since `light*` prefix are already assigned to the bold form

Reply:
Prefix with `lite`?  Add to `color.rb`?